### DISCUSSION TITLE
Running about raspberry

### DISCUSSION THREAD

**Discussion by nfaraji2002** [mudler/LocalAI#269] [Q&A]
Hi,
Thanks for the great package.
Is it possible that we run any model of LocalAI on Raspberry PI 4?
Are the models in LocalAI able to run in a offline mode without internet?
How to customize a model on a specific language?

Thanks in advance,

**Comment by Aisuko** [comment]
Hi, @nfaraji2002. It is good for you to ask questions here. The first and second we can figure out after several months. There many people are interested in it. And for your third question, there is some useful repo you can use like 🤗transfomer, PEFT, ChatLLaMA, etc.

**Comment by mudler** [comment]
@nfaraji2002 rpi should work - however I didn't test it yet, there are container images too that you can try out.

Models are local - no remote connectivity is needed. For fine tuning a model: it really depends on the model family, there are various ways to do so, for instance: https://github.com/ZrrSkywalker/LLaMA-Adapter

**Comment by BenjamenMeyer** [comment]
I'm also curious about this, especially since the Rasberry Pi Foundation released an [AI Kit](https://www.raspberrypi.com/products/ai-kit/) and an [AI Hat+](https://www.raspberrypi.com/products/ai-hat/) for RPi5. Tempting to add it to my collection for this purpose.
